docs/en/02_Developer_Guides/00_Model/06_SearchFilters.md

```diff
@@ -36,6 +36,10 @@ These suffixes can also take modifiers themselves. The modifiers currently suppo
 comparison uses the database's default. For MySQL and MSSQL, this is case-insensitive. For PostgreSQL, this is 
 case-sensitive.
 
+Note that all search filters (e.g. `:PartialMatch`) must be written in title case,
```

Comment:
That'll cause hard-to-debug exceptions - for example, a wrong casing like `MyField:partialmatch` would've satisfied `class_exists('partialmatchFilter')` before, but now throws `ExactMatchFilter does not accept partialmatchFilter as modifiers` (http://stackoverflow.com/questions/15488632/php-class-exists-acting-case-sensitive)

See http://stackoverflow.com/questions/15488632/php-class-exists-acting-case-sensitive